website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -0,0 +1,537 @@
+---
+title: "cross-database macros"
+id: "cross-database-macros"
+---
+
+# Overview
+
+These macros make it easier for package authors (especially those writing modeling packages) to implement cross-database
+compatibility. In general, you should not need these macros in your own dbt project (unless it is a package or you otherwise desire cross-database compatibility).
+
+## All functions (alphabetical)
+- [dbt.any_value](#any_value)
+- [dbt.bool_or](#bool_or)
+- [dbt.cast_bool_to_text](#cast_bool_to_text)
+- [dbt.concat](#concat)
+- [dbt.current_timestamp](#current_timestamp)
+- [dbt.current_timestamp_in_utc](#current_timestamp_in_utc)
```

Comment:
Let's swing back for `type_*` macros (https://github.com/dbt-labs/dbt-utils/issues/598) + `current_timestamp` macros (https://github.com/dbt-labs/dbt-utils/issues/599), once those are ready. (Might be in for v1.2, might be for a future release.) In the meantime, I don't think those need to block the initialization of this page / merging this PR.